acqa-repo1-aws-tf12-part1.tf

```diff
@@ -653,23 +683,23 @@ resource "aws_ecr_repository" "acqa-test-ecr1" {
   image_tag_mutability = "IMMUTABLE"
 
   image_scanning_configuration {
-    scan_on_push = false
+    scan_on_push = true
   }
   tags = {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure public access is disabled for AWS EKS API servers**
 
**Vulnerability**
EKS API Servers set as public may lead to unauthorized access.
 
**Remediation**
To ensure that EKS API servers remain private, it is recommended to use a VPC. Some specific IPs can be whitelisted to let EKS servers be accessed by trusted devices through the internet.

source: accurics-